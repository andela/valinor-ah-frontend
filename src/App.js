import React from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import routes from './routes';

const App = () => (
  <Router>
    <div>
      {
      // eslint-disable-next-line max-len
        routes.map(route => (<Route exact path={route.path} key={route.path} component={route.component} />))
      }
    </div>
  </Router>
);

export default App;
