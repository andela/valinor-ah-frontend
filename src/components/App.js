import React from 'react';
import Button from '../containers/Button';

const App = () => (
  <div className='jumbotron'>
  <h1>Welcome to Author's Haven!</h1>
  <Button />
</div>

);

export default App;
