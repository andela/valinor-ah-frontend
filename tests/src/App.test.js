import React from 'react';
import { shallow } from 'enzyme';

import App from '../../src/App';

test('App snapshot test', () => {
  const component = shallow(<App />);
  // eslint-disable-next-line no-undef
  expect(component).toMatchSnapshot();
});
