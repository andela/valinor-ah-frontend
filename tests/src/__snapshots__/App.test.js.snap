// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<BrowserRouter>
  <div>
    <Route
      component={[Function]}
      exact={true}
      key="/"
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/login"
      path="/login"
    />
  </div>
</BrowserRouter>
`;
